### PR TITLE
Show metadata as string in admin

### PR DESCRIPTION
**Pull request by jhankin**
This fixes https://github.com/Raheem-ai/seed_web/issues/40.  Tested locally and in staging.

### CODE REVIEW

File: config/initializers/rails_admin.rb [hunk 1]
```diff
@@ -25,6 +25,11 @@
   ## To disable Gravatar integration in Navigation Bar set to false
   # config.show_gravatar true
 
+  config.model 'Incident' do
```
Comment (kidbombay) [inline config/initializers/rails_admin.rb]: Typically for rails admin, I make a concern called
incident_admin in concerns/admin and include the concern in the model.

The reason is
1 - Makes all admin model specific related code clearer
2 - rails_admin only gets loaded once, so you can't test it easily on dev. As a concern it's always reloaded.

**Comment by jhankin** [comment]
@kidbombay updated, PTAL

**Comment by jhankin** [comment]
@kidbombay does this look good to you?  Wanna get it merged so @Jlawlzz can use the admin console

**Comment by kidbombay** [comment]
Oh he's looks good. Sorry thought I posted.

**Pull request merged**